README.md

```diff
@@ -281,13 +312,21 @@ Linking.getInitialURL().then((url) => {
 });
 ```
 
+## Troubleshooting
+
+Use the [troubleshooting guide](Troubeshooting.md) to resolve common issues with the Klaviyo React Native SDK.
```

Comment:
might update this link to be absolute once I merge this into master since this readme would be exported to npm

Reply:
🧠 I would not have thought of that.